docs/content/en/_index.md

```diff
@@ -12,44 +12,25 @@ description: "K8ssandra documentation: architecture, configuration, guided tasks
 type: docs
 ---
 
-The K8ssandra documentation covers everything from install details, deployed components, configuration references, and guided outcome-based tasks. Check out the sections at left based on your needs and be sure to leave us a <a class="github-button" href="https://github.com/k8ssandra/k8ssandra" data-icon="octicon-star" aria-label="Star k8ssandra/k8ssandra on GitHub">star</a> on Github!
+The K8ssandra documentation covers everything from install details, deployed components, configuration references, and guided outcome-based tasks. 
 
-{{% alert title="Tip" color="primary" %}}
-If you're impatient, jump right in with our **[install]({{< relref "install" >}})** topics!
-{{% /alert %}}
+Be sure to leave us a <a class="github-button" href="https://github.com/k8ssandra/k8ssandra" data-icon="octicon-star" aria-label="Star k8ssandra/k8ssandra on GitHub">star</a> on Github!
 
-## What is K8ssandra?
+## Features for single- and multi-cluster Kubernetes environments
 
-K8ssandra is a cloud native distribution of Apache Cassandra® (Cassandra) meant to run on Kubernetes. Accompanying Cassandra is a suite of tools to ease and automate operational tasks. This includes metrics, data anti-entropy services, and backup tooling. As part of K8ssandra's installation process all of these components are installed and wired together freeing your teams from having to perform the tedious plumbing of components.
+| K8ssandra v1.4.x      | K8ssandra Operator v1.0.0 |
```

Comment:
I'm thinking we should avoid the version numbers here, try to talk about them more generically.